pkg/rbd/nodeserver.go

```diff
@@ -392,6 +392,7 @@ func (ns *NodeServer) mountVolumeToStagePath(ctx context.Context, req *csi.NodeS
 	}
 
 	opt := []string{"_netdev"}
+	opt = csicommon.ConstructMountOptions(opt, req.GetVolumeCapability())
```

Comment:
you are overriding the previous `"_netdev"` option, that is probably not intentional.

Reply:
it wont override it, `ConstructMountOptions` returns the unique data in opt and req.GetVolumeCapability()